css-four-column-layout/styles.css

```diff
@@ -0,0 +1,44 @@
+body {
+  font-family: Barlow, sans-serif;
+  background-color: #f7f8fa;
+}
+
+h2 {
+  font-size: 30px;
+  margin-top: 10px;
+}
+
+h3 {
+  font-size: 20px;
+  margin-bottom: 30px;
+}
+
+.container {
+  max-width: 1500px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  width: 100%;
+  text-align: center;
+}
+
+.column-fourth {
+  width: 25%;
+  text-align: center;
+  margin-left: 20px;
+  margin-right: 20px;
```

Comment:
Remember not to pollute your layout classes. You should move this to a separate class that you can apply to the element _in addition to_ the `column` class.